src/Parser/DeathKnight/Blood/Modules/Features/BoneShield.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber, formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import StatTracker from 'Parser/Core/Modules/StatTracker';
+
+const SKELETAL_SHATTERING_DR = 0.08;
+const BONE_SHIELD_DR = 0.16;
+
+class BoneShield extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+    statTracker: StatTracker,
+  };
+
+  hasSS = false;
+  hasSD = false;
+
+  boneShieldMitigated = 0;
+  skeletalShatteringMitigated = 0;
+
+
+  on_initialized() {
+    this.hasSS = this.combatants.selected.traitsBySpellId[SPELLS.SKELETAL_SHATTERING_TRAIT.id];
+    this.hasSD = this.combatants.selected.hasTalent(SPELLS.SPECTRAL_DEFLECTION_TALENT.id);
+  }
+
+  get boneShieldAbsorbTooltip() {
+    return formatNumber(this.boneShieldMitigated) + " Bone Shield Absorb<br>";
```

Comment:
Remember to add `this.skeletalShatteringMitigated` for the total amount since `this.boneShieldMitigated` doesn't include the SS part.